app/dashboard/static/js/app/tables/common.js

```diff
@@ -2,9 +2,8 @@
  * Copyright (C) Linaro Limited 2015,2016,2017,2019
  * Author: Matt Hart <matthew.hart@linaro.org>
  * Author: Milo Casagrande <milo.casagrande@linaro.org>
- *
```

Comment:
Looks like this line was removed by accident.  Not a big issue but it creates some noise in the diff.